README.md

```diff
@@ -101,7 +101,7 @@ The value here is an array of objects which contain `serverSelector` and `client
 See more in buffering section below. 
 * `buffer` - If true, preboot will attempt to buffer client rendering to an extra hidden div. 
 See more in buffering section below.
-* `uglify` - If true, the code returned from `getInlineCode(opts)` will be uglified. 
+* `minify` - If true, the code returned from `getInlineCode(opts)` will be minified. 
```

Comment:
Cosmetic change. A couple things like this, but I figured since I am changing so much and this is a major update, shouldn't be that big of a deal. Default for minify is true.